_includes/payment-presentations.md

```diff
@@ -0,0 +1,185 @@
+verification file on the following web path:
+
+https://[DOMAIN_NAME]/.well-known/apple-developer-merchantid-domain-association
+
+THE FILE IN WHATEVER FORMAT WE CHOOSE TO PRESENT IT!!! **(Not a Word file)**
+
+The verification file consists of a hex string which contains a JSON. Do not
+be alarmed if you read the hex data and find this. It is by design.
+
+Opening it is OK, but make sure that you **upload it exactly as it is**. We
+recommend opening it as a text file or something similar, and **not** e.g. in
+Word, as it could result in unwanted formatting changes if you save it. The file
+does not, and should not, have a file extension.
+
+To help you validate that your file looks correct, you can
+[compare it to this][payex-domain-file], which is already present on our server.
+
+**If you are using our iOS SDK**, you need to ensure that the **WebViewBaseURL**
+is set to the exact same domain as where you host the file. Otherwise, it will
+not validate.
+
+This file must be in place before we can activate Apple Pay for you.
+
+### Accepting Donations
+
+Apple Pay provides nonprofit organizations a simple and secure way to accept
+donations. To register your nonprofit organization for Apple Pay, please visit
+[Benevity][benevity-donation-setup].
+
+You’ll be asked to provide basic information about your organization. Note that
+the **Apple Developer Team ID** is an **optional** field, so this is not needed.
+
+When you get your approval from Benevity, you need to share it with Swedbank Pay
+before we can activate Apple Pay for you. You can e-mail it to
+**agreement@swedbankpay.com** together with **Name**, **Organizational** and
+**Customer number**.
+
+### Apple Pay Terms And Conditions
+
+Apple requires Swedbank Pay to identify whether and when you have accessed the
+Apple Pay Platform Web Merchant Terms and Conditions, and to record whether you
+have accepted and agreed to them.
+
+We also need to require you to periodically incorporate updates or amendments to
+the terms of the Apple Pay Web Terms and Conditions, Apple Pay Web Guidelines,
+Apple Pay HI Guidelines, Apple Pay Best Practices Guide, or Apple Marketing
+Guidelines.
+
+Unless you have already accepted as part of signing your agreement with
+Swedbank Pay, we can provide the following links for digital signature in
+[Sweden][apple-pay-tc-sign-sweden] and [Norway][apple-pay-tc-sign-norway].
+
+If you are unable to sign the Apple Pay Web Terms and Conditions in Swedish or
+Norwegian digitally, please use the **supplementary agreement template** (link)
+in English, and e-mail it to **agreement@swedbankpay.com** together with
+**Name**, **Organizational** and **Customer number**. Your acceptance is needed
+before we can activate Apple Pay for you.
+
+{% include iterator.html prev_href="/checkout-v3/"
+                         prev_title="Back to Get Started" %}
+
+## Card
+
+{% include iterator.html prev_href="/checkout-v3/"
+                         prev_title="Back to Get Started" %}
```

Comment:
What's `iterator.html` doing in the middle of the page like this under the `Card` header?

Reply:
The idea was to use the same layout as the checkout v3 sequence diagrams page, where each integration section would link to the corresponding diagram, and have a quick way back, so you don't have to scroll all the way down. Pick an instrument you want to read about, pop back to the matrix as easy as possible.

Reply:
Aha, I see.